cli/jobs.py

```diff
@@ -1,14 +1,102 @@
 import click
+from rich.console import Console
+from rich.table import Table
+from rich.text import Text
+from rich import box
+from json import dumps as json_dumps
 
 
-@click.group("jobs", invoke_without_command=True)
-def jobs():
+@click.command(short_help="Info about Jobs")
+@click.option("-a", "--all", is_flag=True, help="List all jobs")
+@click.pass_context
```

Comment:
can you move this down? This should be the last decorator after all options, arguments..just for style and formatting

Reply:
You mean `Line 10`?

Reply:
I mean interchange line 11 and 12, options should be above pass_context